tests/sentry/models/test_groupreaction.py

```diff
@@ -0,0 +1,190 @@
+import pytest
+from django.db import IntegrityError
+
+from sentry.models.groupreaction import GroupReaction, GroupReactionType
+from sentry.testutils.cases import TestCase
+
+
+class GroupReactionTest(TestCase):
```

Comment:
Up to you, but you don't necessarily need to keep tests around to verify the constraints here